src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -19,6 +19,11 @@
         </div>
         <h5 *ngIf="workItem.id" class="detail-id pull-left" id="wi-detail-id">{{workItem.id}}</h5>
         <span class="pficon-close detail-close" (click)="closeDetails()"></span>
+        <div class="pull-right created-wrap">
+          <span title="{{workItem.attributes['system.created_at'] | date:'fullDate'}}">
```

Comment:
Try adding this title using the pipe so that we can use the pipe with that hover feature every where.